src/main/java/au/gov/digitalhealth/terminology/amtflatfile/Amt2FlatFile.java

```diff
@@ -183,6 +197,60 @@ public void execute() throws MojoExecutionException, MojoFailureException {
 		}
 	}
 
+    private void validateOutputPath(String outputPath, String expectedMimeType) {
+        try {
+            Path path = Paths.get(outputPath);
+
+            if (Files.isSymbolicLink(path)) {
+                throw new SecurityException("The input ZIP file must not be a symlink for security reasons");
```

Comment:
Should this be "The **output** ZIP file must not be a symlink for security reasons"

Reply:
yes, you're right, thanks for spotting that